Kickstarter-iOS/Views/Controllers/ProjectPamphletViewControllerTests.swift

```diff
@@ -316,60 +271,10 @@ internal final class ProjectPamphletViewControllerTests: TestCase {
     }
   }
 
-  // MARK: - Native Checkout Disabled or Undefined
-
-  func testLoggedOut_LiveProject_NativeCheckout_Disabled() {
-    let config = Config.template
-      |> \.features .~ [Feature.nativeCheckout.rawValue: false]
-
-    let language = Language.en
-    let device = Device.phone4_7inch
-
-    let liveProject = self.project
-      |> Project.lens.stats.convertedPledgedAmount .~ 1_964
-
-    // All we want to see here is that the pledge CTA button is hidden
-
-    withEnvironment(
-      apiService: MockService(fetchProjectResponse: liveProject),
-      config: config, currentUser: nil, language: language
-    ) {
-      let vc = ProjectPamphletViewController.configuredWith(projectOrParam: .left(liveProject), refTag: nil)
-      _ = traitControllers(device: device, orientation: .portrait, child: vc)
-
-      FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
-    }
-  }
-
-  func testLoggedOut_LiveProject_NativeCheckout_Undefined() {
-    let config = Config.template
-      |> \.features .~ [:]
-
-    let language = Language.en
-    let device = Device.phone4_7inch
-
-    let liveProject = self.project
-      |> Project.lens.stats.convertedPledgedAmount .~ 1_964
-
-    // All we want to see here is that the pledge CTA button is hidden
-    withEnvironment(
-      apiService: MockService(fetchProjectResponse: liveProject),
-      config: config, language: language
-    ) {
-      let vc = ProjectPamphletViewController.configuredWith(projectOrParam: .left(liveProject), refTag: nil)
-
-      let (parent, _) = traitControllers(device: device, orientation: .portrait, child: vc)
-      parent.view.frame.size.height = device == .pad ? 2_300 : 1_800
-
-      FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
-    }
-  }
-
-  // MARK: - Native Checkout Enabled, Error fetching project
+  // MARK: - Error fetching project
 
-  func testProjectPamphletViewController_NativeCheckoutEnabled_ErrorFetchingProject() {
+  func testProjectPamphletViewController() {
```

Comment:
Should we rename this to `testProjectPamphletViewController_ErrorFetchingProject`? I think it's more descriptive to what we are testing here.